### DISCUSSION TITLE
GCode sender 5 axis

### DISCUSSION THREAD

**Discussion by yushiromarcel** [fra589/grbl-Mega-5X#259] [Q&A] [answered]
HEllo Gauthier, 
I wonder which G code sender I can use to controle GRBL MEGA 5X with a PC.
Because Universal Gcode Sender seems to manage only 3 axis (on the jog controller panel for exemple)

What tool do you use ? Did you tried GRBLGRU ? Other tool ?

Thanks !

**Comment by fra589** [comment, marked as answer]
Hi @yushiromarcel,  
  
Since I did not find this rare gem, in 2018 I began to wrote my own Grbl sender [cn5X++](https://github.com/fra589/cn5X), specially for grbl-Mega-5X. It still in active development, but I already use it for real milling on my CNC milling.  
I work and develop under Linux, but I have feedback from users who use it successfully on Windows.  
You will find it here : https://github.com/fra589/cn5X  
I haven't had time to write the user documentation yet, but the installation procedure on Windows is in the [wiki](https://github.com/fra589/cn5X/wiki).  
  
@++;  
Gauthier.
